setup.sh

```diff
@@ -1,75 +1,133 @@
+#!/bin/bash
 SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
 HAS_GIT=$(which git)
-NOTABLE_DIR=~/.notable
```

Comment:
I don't think this should have been removed?

Reply:
Sure, thanks! I'll fix it.

Reply:
I added it on line **118** and **122** because using `sudo` changes the path of `~/`.